README.md

```diff
@@ -229,6 +233,39 @@ AND
 msauth://code/<broker-redirect-uri-in-url-encoded-form>/
 ex: msauth://code/mytestiosapp%3A%2F%2Fcom.mycompany.myapp and msauth://code/mytestiosapp%3A%2F%2Fcom.mycompany.myapp/  
 ```
+### Brokered Authentication for Android
+
+If your app requires conditional access or certificate authentication support, you must set up your AuthenticationContext and redirectURI to be able to talk to the Azure Authenticator app OR Company Portal. Make sure that your Redirect URI and application's bundle id is all in lower case.
```

Comment:
if your app is susceptible of requiring? I mean this can be decided by the Azure AD admin? not the developer of the app?

Reply:
app itself can have conditional access forced on it too via tenant admin. I will update the doc to be clearer